src/stores/Widgets/CatalogWidget/CatalogWidgetStore.ts

```diff
@@ -311,14 +344,27 @@ export class CatalogWidgetStore {
         this.angleMin = CatalogWidgetStore.MinAngle;
     }
 
+    /**
+     * Set the maximum orientation value
+     * @param max - number : max degree of orientation
```

Comment:
Thanks for adding the documentation.
Please remove all the types in the documentation because TSDoc will handle this part automatically.